src/cli/argument-parser.js

```diff
@@ -112,6 +112,7 @@ export default class CLIArgumentParser {
             .option('--disable-page-reloads', 'disable page reloads between tests')
             .option('--dev', 'enables mechanisms to log and diagnose errors')
             .option('--qr-code', 'outputs QR-code that repeats URLs used to connect the remote browsers')
+            .option('--ignore-uncaught-errors', 'ignore uncaught errors')
```

Comment:
I propose to use a more strict message: `ignore uncaugt errors and unhandled promise rejection happens during test exection`.
Need to discuss with @VasilyStrelyaev

Reply:
run through correctors:
```
            .option('--ignore-uncaught-errors', 'ignore uncaught errors and unhandled promise rejections, which occur during test execution')
```

Reply:
The phrase is checked with correctors, it is ok.